config/initializers/active_record_extensions.rb

```diff
@@ -1,5 +1,10 @@
+require 'host_mix'
+require 'has_many_common'
+require 'strip_whitespace'
+require 'parameterizable'
+
 class ActiveRecord::Base
```

Comment:
TBH, this initializer predates the introduction of ApplicationRecord in rails. Probably most if not all of the extensions actually belong there.